.travis.yml

```diff
@@ -0,0 +1,41 @@
+language: bash
+
+group: bluezone
+
+dist: xenial
+
+os: linux
+
+sudo: required
+
+matrix:
+    include:
+      - os: linux-ppc64le
+      - os: linux-s390
```

Comment:
for public Github Travis we can just build x86, as we won't use this build to push out images.  So we can remove lines 13/14.